go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/google/go-cmp v0.5.7
 	github.com/gophercloud/gophercloud v0.22.0
 	github.com/hooklift/gowsdl v0.5.0
-	github.com/infobloxopen/infoblox-go-client/v2 v2.1.1
+	github.com/infobloxopen/infoblox-go-client/v2 v2.1.2-0.20220401192150-b6079a2c974a
```

Comment:
This doesn't look like an official release, is there some context behind this?

Reply:
There was no official release of go-client so far, after v2.1.1, thus I had to use this intermediate version, which includes the new feature which will be needed in the nearest future. Current full code is here: https://github.com/skudriavtsev/external-dns-1/commits/client-cert-auth This PR contains just a portion to fix particular issue.

Thanks for the comment. Anyway, the version v2.1.2-0.20220401192150-b6079a2c974a is not right, it contains a typo. I have bumped it up already.